page-optimize.php

```diff
@@ -31,6 +31,8 @@
 	exit;
 }
 
+require_once __DIR__ . '/settings.php';
```

Comment:
The settings don't need to be conditionally loaded. Let's load them unconditionally.